src/components/InputTimeInterval.vue

```diff
@@ -1,55 +1,64 @@
 <template lang="pug">
 div
   div
-    b-alert(v-if="mode == 'range' && invalidDaterange", variant="warning", show)
+    b-alert(v-if="invalidDaterange", variant="warning", show)
       | The selected date range is invalid. The second date must be greater or equal to the first date.
-    b-alert(v-if="mode == 'range' && daterangeTooLong", variant="warning", show)
+    b-alert(v-if="daterangeTooLong", variant="warning", show)
       | The selected date range is too long. The maximum is {{ maxDuration/(24*60*60) }} days.
 
-  div.d-flex.justify-content-between.align-items-end
```

Comment:
Any particular thought behind removing the flex and right-side content?

Reply:
By right side content you mean the update button with the sync icon?
I think this button is not required anymore. Since the timeline is updated on duration selection automatically, or by clicking the new "apply" button, which is no right next to the date range inputs.

Reply:
The "apply" button only works on the range, not the time interval, so it's not really functionally equivalent. It was meant to be a way to refresh without reloading the entire page.

Reply:
Ah I see. I never understood the difference between the two update buttons :) thought they solve the same problem ... But now I understand. I have re-added it again. What do you think?

Reply:
Ping :)

Reply:
Hi @ErikBjare did you have time to look at it? Did I miss something? For me, the proposed changes in this PR would help me in daily business, and I hope/think others as well.

Reply:
Getting to it now! Will merge today :)